services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -153,26 +159,31 @@ export default class ApplyPage extends React.Component<Props> {
             }) => (
               <form onSubmit={handleSubmit}>
                 <div className="g">
-                  <div className="sel">
-                    <label
-                      htmlFor="FeedbackForm-${this.props.name}"
-                      className="txt-l txt-l--sm"
-                    >
-                      Prefix{' '}
-                    </label>
-                    <div
-                      className="sel-c sel-c--thin"
-                      style={{ marginRight: 14 }}
-                    >
-                      <select className="sel-f sel-f--thin">
-                        <option>Mr</option>
-                        <option>Mrs</option>
-                        <option>Miss</option>
-                        <option>Mx</option>
-                      </select>
+
+                  <div className="g--6 m-b300"
+                       style={{
+                         display: 'flex',
```

Comment:
It would be good to switch this to Emotion so that these get vendor-prefixed as necessary.

Reply:
oops, I am used to autoprefixer in my builds and spaced out on this - thx!